### PR TITLE
Add up to 4 ICEs

### PR DESCRIPTION
**Pull request by fanninpm**
NOTE: 64450 and 66922 have `git clone` commands. Should they be removed?

### CODE REVIEW

File: ices/64450.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+cleanup() {
+    rm -r "$tmp"
+}
+
+trap cleanup EXIT
+
+cd "$tmp"
+
+git clone https://github.com/thiolliere/debug-inconsistence.git && cd debug-inconsistence/runtime
```
Comment (Alexendoo) [inline ices/64450.sh:19]: Yeah I don't think ones should be added that are not self contained

File: ices/66805.sh [hunk 2]
```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+tmp="$(mktemp -d)"
```
Comment (Alexendoo) [inline ices/66805.sh:3]: glacier runs in a temp directory and handles cleanup, so mktemps can be removed

Comment (fanninpm) [reply ices/66805.sh:3]: This is a holdover from old code (fixed/59191.sh). Thanks for telling me!

File: ices/64450.sh (hunk 1 above)
Comment (fanninpm) [reply ices/64450.sh:19]: Mind if I _make_ them (64450, 66003, and 66922) self-contained?

**Comment by Alexendoo** [comment]
What are your thoughts on having crate dependencies in ICEs @JohnTitor? (quote, syn, etc)

**Pull request merged**